gnosis/signer.js

```diff
@@ -0,0 +1,18 @@
+const Safe = require('@safe-global/safe-core-sdk')
+const { SafeEthersSigner, SafeService } = require('@safe-global/safe-ethers-adapters')
+const ethAdapter = require('./adapter')
+
+async function getGnosisSigner(DEPLOYER_SAFE){
```

Comment:
We can rename DEPLOYER_SAFE to follow camel case to signify that it's a parameter.

Reply:
done in latest commit